CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file.
 <!--
 Populate these categories as PRs are merged to `main`. When a release is cut,
 copy to its corresponding section, deleting empty sections if any.
+Remember to update the hyperlinks at the bottom.
```

Comment:
This is a great reminder, thanks for including it directly in the changelog comments!